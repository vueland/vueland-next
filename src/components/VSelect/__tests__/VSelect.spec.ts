import { mount, VueWrapper } from '@vue/test-utils'
import { VSelect } from '../index'
import 'regenerator-runtime/runtime'

describe('VSelect', () => {
  let mountFunction: (options?: any) => VueWrapper<any>

  beforeEach(() => {
    mountFunction = (options = {}) =>
      mount(VSelect, {
        ...options,
        global: {
          provide: {
            $options: null,
          },
        },
      })
  })

  it('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
