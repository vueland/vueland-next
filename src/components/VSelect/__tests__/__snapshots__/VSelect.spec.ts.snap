// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VSelect should mount component and match snapshot 1`] = `
<div class="v-input">
  <div class="v-input__slot"
       style="color: rgb(33, 150, 243); caret-color: #2196f3;"
  >
    <div class="v-input__select-slot">
      <label class="v-label v-label--on-field"
             style="left: 0px; position: absolute; color: rgb(33, 150, 243); caret-color: #2196f3;"
      >
      </label>
      <div class="v-select">
        <input class="v-select__input"
               style="color: rgb(66, 66, 66); caret-color: #424242;"
        >
      </div>
    </div>
    <div class="v-input__status">
      <transition-stub>
      </transition-stub>
    </div>
  </div>
</div>
`;
